chain/jsonrpc/src/test_utils.rs

```diff
@@ -0,0 +1,67 @@
+use actix::Actor;
+
+use near_client::{ClientActor, ViewClientActor};
+use near_network::test_utils::convert_boot_nodes;
+use near_store::test_utils::create_test_store;
+
+use actix::actors::mocker::Mocker;
+use near_network::types::{NetworkViewClientMessages, NetworkViewClientResponses};
+use near_network::{NetworkClientResponses, NetworkConfig, PeerManagerActor};
+use near_primitives::block::GenesisId;
+use near_primitives::borsh::maybestd::sync::atomic::AtomicUsize;
+use near_primitives::network::PeerId;
+use std::sync::atomic::Ordering;
+use std::sync::Arc;
+
+type ClientMock = Mocker<ClientActor>;
+type ViewClientMock = Mocker<ViewClientActor>;
+
+#[allow(dead_code)]
+pub fn make_peer_manager(
```

Comment:
What is the reason for putting it inside the `near-jsonrpc` crate instead of `near-jsonrpc-test-utils`? Can we move it?

Reply:
I wanted to avoid code duplication: `chain/jsonrpc/test-utils/src/lib.rs` and `chain/jsonrpc/tests/test_utils/mod.rs`.
@frol Hmm, maybe I need to create a new package.

Reply:
Why does it belong to `jsonrpc` at all? Can we put it in near-network test utils?

Reply:
I moved it out.